lib/pages/main/wallet_contents/settings/wallet_connect/wallet_connect.dart

```diff
@@ -56,16 +55,11 @@ class _AboutWalletState extends State<WalletConnectSettings> {
   }
 
   List<String> getMethods(SessionData sessionData) {
```

Comment:
it would be better to get the chain from the chainid instead of defining another constant:

eturn chainId.split(':')[0];

Also, I saw they are some useful methods in 'package:reown_sign/reown_sign.dart';

class NamespaceUtils  that we can use